robowflex_tesseract/scripts/ur5_custom_planning.cpp

```diff
@@ -0,0 +1,156 @@
+/* Author: Carlos Quintero Pena */
+
+#include <robowflex_library/detail/ur5.h>
+#include <robowflex_library/io/visualization.h>
+#include <robowflex_library/log.h>
+#include <robowflex_library/planning.h>
+#include <robowflex_library/robot.h>
+#include <robowflex_library/scene.h>
+#include <robowflex_library/trajectory.h>
+#include <robowflex_library/util.h>
+#include <robowflex_tesseract/conversions.h>
+#include <robowflex_tesseract/trajopt_planner.h>
+
+/* \file ur5_custom_planning.cpp
+ * Simple demonstration of how to use the TrajOptPlanner with the UR5 with custom terms in the optimization.
+ * This customized trajopt planner has end-effector pose constraints at specific timesteps of the trajectory.
+ */
+
+namespace robowflex
```

Comment:
the custom class doesn't need to be in the robowflex namespace, you can just have it in the global